includes/class-simple-local-avatars.php

```diff
@@ -326,6 +326,20 @@ public function get_user_id( $id_or_email ) {
 		return $user_id;
 	}
 
+	/**
+	 * Get the local avatar user meta.
+	 *
+	 * @param int $user_id User ID.
+	 * @return array Array with avatar data.
+	 */
+	public static function get_user_local_avatar( $user ) {
```

Comment:
We use the variable `$user_id` but the only variable passed in is `$user`. I think that just needs updated:

```suggestion
	public static function get_user_local_avatar( $user_id ) {
```